velox/connectors/hive/storage_adapters/gcs/tests/GCSTestbench.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <boost/process.hpp>
+#include <gmock/gmock-matchers.h>
+#include <gmock/gmock-more-matchers.h>
+#include <google/cloud/storage/client.h>
+#include "gtest/gtest.h"
+
+#include "velox/common/config/Config.h"
+#include "velox/connectors/hive/storage_adapters/gcs/GCSUtil.h"
+#include "velox/exec/tests/utils/PortUtil.h"
+
+namespace bp = boost::process;
+namespace gc = google::cloud;
+namespace gcs = google::cloud::storage;
+
+namespace facebook::velox::filesystems {
+
+class GCSTestbench : public testing::Environment {
```

Comment:
let's rename this to something more descriptive and intuitive. "bench" implies benchmarking which is not the intent.

Reply:
Named it as `GcsEmulator`.